packages/ng/icon/icon.component.scss

```diff
@@ -1,4 +1,4 @@
-@use '@lucca-front/icons/src/main';
+// @use '@lucca-front/icons/src/main';
```

Comment:
delete?

Reply:
Damn, I forget about this one:

```bash
✘ [ERROR] Could not resolve "../../icons/src/commons||file://cdn.lucca.fr/lucca-front/icons/font/lucca-icons.woff" [plugin angular-css-resource]

    icon/icon.component.scss:14:91:
      14 │ ...f2"), url("../../icons/src/commons||file://cdn.lucca.fr/lucca-f...
         ╵              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  Preprocessor stylesheets may not show the exact file location of the error.
```

It should be fixed before merging this PR